homeassistant/components/todoist/calendar.py

```diff
@@ -296,6 +296,7 @@ def __init__(
         )
         self._cal_data = {}
         self._name = data[CONF_NAME]
+        self._attr_unique_id = data.get(CONF_ID)
```

Comment:
If this is a custom project configured in yaml by the user the unique_id will be `None`, otherwise it will return the stable ID of the project from the api.

Reply:
Good.